src/routes/authRouter.js

```diff
@@ -17,6 +19,6 @@ router.route("/login").post(login);
 router.route("/logout").post(logout);
```

Comment:
I know this isn't new to this PR, but I wasn't sure why some of the routes look like `router.route("/logout").post(logout)` instead of just `router.post("/logout", logout)`?

Reply:
Yeah this is on me, some inconsistencies slip through sometimes. Thanks for pointing it out, will fix on the next commit.